src/table/index.js

```diff
@@ -212,19 +228,56 @@ class TableField extends Component {
       schema,
       uiSchema,
       registry,
-      version: "1",
+      version: "1"
     };
     return <InsertModal {...attr} />;
   };
+  checkMyRow(isTableExpandable) {
```

Comment:
Is checkMyRow checking to see if the rowIsExpandable?

Reply:
Yes